### PR TITLE
[marketplace] add providers tabs

### PR DESCRIPTION
**Pull request by markkaylor**
What does it do?
Adds tabs for the marketplace to support Plugins and Providers

Why is it needed?
To make it easy to distinguish between plugins and providers

How to test it?
Go the Markateplace page and select the different tabs:

You should only see plugins on the plugins tab and providers on the providers tab

Note:
I realized we didn't consider how to handle installed providers so aren't marked as such in this PR.

@remidej I think we should discuss and handle it in another PR based off this one so we can keep moving forward with the search as well.

### CODE REVIEW

File: packages/core/admin/admin/src/pages/MarketplacePage/index.js [hunk 1]
```diff
@@ -225,18 +231,31 @@ const MarketPlacePage = () => {
...
-              {searchResults.map(plugin => (
-                <GridItem col={4} s={6} xs={12} style={{ height: '100%' }} key={plugin.id}>
-                  <PluginCard
-                    plugin={plugin}
-                    installedPluginNames={installedPluginNames}
+            <TabGroup label="Some stuff for the label" id="tabs" variant="simple">
+              <Box paddingBottom={4}>
+                <Tabs>
+                  <Tab>Plugins ({searchResults.length})</Tab>
+                  <Tab>Providers ({marketplaceProvidersResponse.data.length})</Tab>
```
Comment (remidej) [inline packages/core/admin/admin/src/pages/MarketplacePage/index.js:238]: These may need to have translation, or at least it's worth checking with Emilie

File: packages/core/admin/admin/src/pages/MarketplacePage/index.js [hunk 2]
```diff
@@ -225,18 +231,31 @@ const MarketPlacePage = () => {
               )}
             />
           ) : (
-            <Grid gap={4}>
-              {searchResults.map(plugin => (
-                <GridItem col={4} s={6} xs={12} style={{ height: '100%' }} key={plugin.id}>
-                  <PluginCard
-                    plugin={plugin}
-                    installedPluginNames={installedPluginNames}
+            <TabGroup label="Some stuff for the label" id="tabs" variant="simple">
```
Comment (remidej) [inline packages/core/admin/admin/src/pages/MarketplacePage/index.js:234]: Don't we have anyhting to put here as a label?

**Review by remidej — CHANGES_REQUESTED** [review]
We'll also have to remember to add tests in another PR

**Comment by markkaylor** [comment]
> We'll also have to remember to add tests in another PR
What test case do we need to check here? For some reason I thought even with these changes the snapshot test was enough... 🤔

**Review by remidej — COMMENTED** [review]
All the code changes look good to me, thanks!

For the tests: first we'd have to update the snapshot (the checks currently fail because of it).

But I was thinking we could add a smarter test for the tabs:

- check that by default the plugins tab is active, and we see a list of plugins
- simulate a click on providers
- check that the providers tab is now active, and we see a list of providers

**Pull request merged**